### PR TITLE
Added hub docs link within container

### PR DESCRIPTION
**Pull request by FrenchBen**
Added link to the docs for the specified image within the container window - Allowing for easier lookup of documentation and how-to for that particular container.

Signed-off-by: FrenchBen <me(at)frenchben.com>

### CODE REVIEW

**Comment by FrenchBen** [comment]
@wkoszek I believe this is what we discussed.

kite-docs

**Comment by wkoszek** [comment]
This looks very good. Can you show the screen shot when the right tab is
set to "Settings" ?

I proposed (?) icon since the box+arrow button was already used here (Home
tab when the container is started):

[image: Inline image 1]


On Fri, Aug 21, 2015 at 11:46 AM, FrenchBen <notifications@github.com>
wrote:

> @wkoszek <https://github.com/wkoszek> I believe this is what we discussed.
>
> [image: kite-docs]
> <https://cloud.githubusercontent.com/assets/3965324/9416387/5ba3e8d0-4813-11e5-9fe6-130f5d3c86d3.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kitematic/kitematic/pull/965#issuecomment-133528248>.
>



-- 
Wojciech A. Koszek
wojciech@koszek.com
http://www.koszek.com/

**Comment by FrenchBen** [comment]
Yup the '?' will have to be added in to the image list (easy change) - For right now I re-used the same icon found on the actual image. 
docs-settings

**Comment by wkoszek** [comment]
That looks good.

On Fri, Aug 21, 2015 at 3:43 PM, FrenchBen <notifications@github.com> wrote:

> Yup the '?' will have to be added in to the image list (easy change) - For
> right now I re-used the same icon found on the actual image.
> [image: docs-settings]
> <https://cloud.githubusercontent.com/assets/3965324/9420522/75a4300c-4834-11e5-9b03-b6eb9298e073.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kitematic/kitematic/pull/965#issuecomment-133581531>.
>



-- 
Wojciech A. Koszek
wojciech@koszek.com
http://www.koszek.com/

File: src/components/ContainerDetailsSubheader.react.js [hunk 1]
```diff
@@ -119,6 +129,10 @@ var ContainerDetailsSubheader = React.createClass({
       action: true,
       disabled: this.disableTerminal()
     });
+    var docsActionClass = classNames({
+      action: true,
+      disabled: false
+    })
```
Comment (jeffdm) [inline src/components/ContainerDetailsSubheader.react.js]: Missing semicolon

**Comment by jeffdm** [comment]
This is neat! Thanks Ben! One small comment then LGTM.

**Comment by FrenchBen** [comment]
Done and good catch!  We need to get lint in place ;)

**Comment by mchiang0610** [comment]
@FrenchBen Tried it, and got this

image

>> [87476:0827/134123:INFO:renderer_main.cc(200)] Renderer process started
>> [87476:0827/134133:WARNING:channel.cc(549)] Failed to send message to ack remove remote endpoint (local ID 1, remote ID 1)
>> [87476:0827/134133:WARNING:channel.cc(549)] Failed to send message to ack remove remote endpoint (local ID 2147483648, remote ID 2)

**Comment by jeffdm** [comment]
LGTM!

**Pull request merged**